pkg/apis/v1/nodeclaim_status.go

```diff
@@ -0,0 +1,68 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"github.com/awslabs/operatorpkg/status"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	ConditionTypeLaunched             = "Launched"
+	ConditionTypeRegistered           = "Registered"
+	ConditionTypeInitialized          = "Initialized"
+	ConditionTypeEmpty                = "Empty"
+	ConditionTypeDrifted              = "Drifted"
```

Comment:
I think you need to update this with the ConsolidateAfter changes, but you could consider doing that as a followup

Reply:
I added those changes